src/pages/Profile.tsx

```diff
@@ -22,11 +22,21 @@ const Profile: React.FC = () => {
 
 	const email = user?.email ?? ''
 
+	// TODO: add the motto to the database
```

Comment:
Part of this issue is also adding "default" avatar in case of no avatar. I haven't see that?

Reply:
Yes. I have an icon when the picture doesn't appear.

Reply:
.. just delete the access to the random user.
![image](https://user-images.githubusercontent.com/58363173/111781873-a6479080-88b8-11eb-94fc-373b2ec63c7d.png)

Reply:
Awesome! :3 Sorry, I somehow overlooked it.